cocos/ui/UISlider.cpp

```diff
@@ -533,7 +533,7 @@ void Slider::adaptRenderers()
 
 Size Slider::getVirtualRendererSize() const
 {
-    return _barRenderer->getContentSize();
+    return _barRenderer->getOriginalSize();
```

Comment:
Maybe I should change all the `getVirtualRendererSize` to return the original size instead of the content size. Because of the content size may equals to the preferred size.